classes/class04/burgerjoint/routes/index.js

```diff
@@ -0,0 +1,169 @@
+var mongoose = require('mongoose');
+var path = require('path');
+var Ingredient = require('./../models/ingredientModel.js');
+var Order = require('./../models/orderModel.js');
+// var Order = require(path.join(__dirname,'../models/orderModel.js'));
+
+var index = {};
+
+index.home = function(req, res){
+  // res.render('home');
+  res.render("home");
+};
+
+index.listIngredients = function (req, res) {
```

Comment:
You definitely should have separated things out into multiple routes like "kitchen" and "ingredients." Doing it all in just index is very disorganized